themes/digital.gov/layouts/partials/core/head.html

```diff
@@ -117,7 +117,10 @@
     content="{{- $.Params.summary | default $.Site.Params.description | markdownify -}}"
   />
   <meta property="og:type" content="{{- $.Scratch.Get "pagetype" -}}" />
-  <meta property="og:url" content="{{- .Permalink -}}" />
+  <meta
+    property="og:url"
+    content="{{ .Site.Params.hostName }}{{- .Permalink -}}"
```

Comment:
## Communities

Returns `og:url` value of `https://digital.gov//localhost:1313/communities/`.

![image](https://github.com/GSA/digitalgov.gov/assets/3385219/7335f749-a6f4-4aa5-bf7e-a0088a6aef09)

In Cloud Pages preview I see a link that leads to 404:

```
https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/nl-fix-og-url/communities/
```

It should be:
```
https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/nl-fix-og-url/communities/
```

Reply:
@mejiaj Both of the URL's are the same in the examples above. For some reason I do not see `https://www.digital.gov` set on my local, still uses `localhost:1313`, could be a caching issue with your code suggestions.

1. Updated `baseURL: https://www.digital.gov` in `config.yml`
2. And set:
```html
<meta property="og:url" content="{{- .Permalink | absURL -}}"/>
```